recipes/haphpipe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: haphpipe
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+
+source:
+  url: https://github.com/gwcbi/haphpipe/archive/v{{ version }}/haphpipe-{{ version }}.tar.gz
+  sha256: c536d3db94cf7c2ef6079249f5017f0f442ecff4d43c3441180760cf4ad8b87e
+  
+requirements:
+
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python=3.7
+    - future 
+    - pyyaml 
+    - biopython 
+    - seqtk 
+    - bowtie2 
+    - bwa 
+    - flash 
+    - freebayes 
+    - mummer 
+    - picard 
+    - trimmomatic 
+    - samtools=1.9 
+    - gatk=3.8 
+    - spades 
+    - blast 
+    - sierrapy 
+    - mafft 
+    - modeltest-ng 
+    - raxml 
+    - sra-tools
+
+test:
+  commands:
+    - haphpipe -h
+
+about:
+  home: https://github.com/gwcbi/haphpipe
+  license: 'open source'
```

Comment:
Can you clarify what sort of "open source" license this has? Ideally it's something like MIT or GPL, but technically we need to know (and they probably need to put a copy of the license together with the source code).

Reply:
Thank you for reviewing this! I am following up with my PI now about the license. In the meantime, could you please explain the changes made in your recent commits, particularly in removing the host requirements?

Reply:
The host requirements aren't actually needed for installation, just for running things, so my changes just speed up building a bit.

Reply:
BTW, thanks for the PR!

Reply:
The changes to `python=3.7` are just to future proof things a bit. We're likely to start supporting python 3.8 within the next month or so and I'd prefer to not have to modify so many recipes that artificially restrict python compatibility to specific ranges.

Reply:
Thank you for the clarifications! 

We'd like to use the MIT License.

Reply:
@dpryan79 We have addressed the License issue and updated our repository accordingly. I have committed and tested the updated recipe now!